presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -90,6 +94,18 @@ public Symbol symbol(String name, Type type)
         return symbol;
     }
 
+    public WindowNode window(WindowNode.Specification specification, Map<Symbol, WindowNode.Function> functions, PlanNode source)
```

Comment:
Maybe create a method with all the parameters, and then overload it for usability.